src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -155,6 +155,19 @@ typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
 };
 
 /**
+ * Flags for options of a gRPC call
+ *
+ */
+typedef NS_OPTIONS(NSUInteger, GRPCCallFlags) {
+  /** Signal that the call is idempotent */
+  GRPCFlagIdempotentRequest = 0x00000010,
```

Comment:
For `NS_OPTIONS` and `NS_ENUM`, it's important that the type name is a prefix of each value's name. Doing it lets the Swift translator present it to Swift developers without the prefix. E.g. this one in Swift would be `GRPCCallFlags.idempotentRequest`.